demo/midgard-node/src/services/always-succeeds.ts

```diff
@@ -4,52 +4,109 @@ import {
   applyDoubleCborEncoding,
   MintingPolicy,
   mintingPolicyToId,
+  Script,
   SpendingValidator,
   validatorToAddress,
 } from "@lucid-evolution/lucid";
 import { NodeConfig, NodeConfigDep } from "@/config.js";
 
 export const makeAlwaysSucceedsServiceFn = (nodeConfig: NodeConfigDep) =>
   Effect.gen(function* () {
-    const spendingCBOR = yield* pipe(
+    const stateQueueSpendingCBOR: string = yield* pipe(
       Effect.fromNullable(
         scripts.default.validators.find(
-          (v) => v.title === "always_succeeds.spend.else",
+          (v) => v.title === "always_succeeds.spend_queue.else",
```

Comment:
These accessors will have the same names as their validator counterparts. If you rename the validators and generate the `plutus.json` file (with `aiken build`), you can see them there.
```suggestion
          (v) => v.title === "always_succeeds.state_queue_spend.else",
```